07-sampling.Rmd

```diff
@@ -1235,15 +1505,15 @@ if (is_latex_output()) {
 }
 ```
 
-```{r echo=FALSE, purl=FALSE, results="asis"}
+```{r echo=FALSE, purl=FALSE, results="asis", eval = FALSE}
```

Comment:
@avaldivi6 I'm pretty sure this does need to run just in case things move around?